notebooks/MAST/JWST/JWST_EngineeringDatabase_Retreival/JWST_EDB_Retrieval.ipynb

```diff
@@ -0,0 +1,560 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# JWST Engineering Data Retrieval"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial will show you how to retrieve JWST engineeering data and use it in the context of a python session. The [Engineering Data](https://outerspace.stsci.edu/pages/viewpage.action?spaceKey=DraftMASTDOCS&title=.Engineering+Data+v1.3) chapter of the [JWST Archive Manual](https://outerspace.stsci.edu/pages/viewpage.action?spaceKey=DraftMASTDOCS&title=.JWST+Archive+Manual+v1.2) describes how data from thousands of engineering telemetry points on JWST are stored in the Engineering Database in the form of timeseries. These data may be searched by means of an identifier, or **mnenomic**. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Some quantities of interest require more than one mnemonic (a *tuple*) for meaningful analysis. This tutorial illustrates how to retrieve a tuple of mnenomics and visualize the result. In the following example timeseries will be retrieved for mnemonics: `SA_ZADUCMDX` and `SA_ZADUCMDY` which are the *x-* and *y-* positions of the JWST [Fine Steering Mirror](https://jwst-docs.stsci.edu/jwst-observatory-hardware/jwst-telescope). \n",
+    "\n",
+    "Note that this folder includes a companion script; after completing the tutorial, this offers a compact, customizable way to download the data.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "\n",
+    "<span style=\"color:black\">\n",
+    "    JWST Engineering data retrieval is restricted to authorized persons prior to the completion of instrument commissioning.  \n",
+    "    \n",
+    "</span>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The workflow consists of: \n",
+    "* [Setup](#Setup)\n",
+    "    * [Fetching your MAST.auth token](#Fetching-a-MAST.auth-Token)\n",
```

Comment:
This anchor link does not work for me (all the others do), must be missing the anchor ID below somewhere?